readme.md

```diff
@@ -67,14 +73,12 @@ export default {
 }
 ```
 
-This imports the `WebAssembly.Module` directly instead of being wrapped in a promise.
+This returns an instantiated module, rather than the binary:
```

Comment:
It doesn't yet return an instantiated module.  It returns `WebAssembly.Module` like it would asynchronously, but it can also be instantiated synchronously:

```js
const inst = new WebAssembly.Instance(mod, { ... })
```